controllers/kuadrant_controller.go

```diff
@@ -493,38 +494,71 @@ func (r *KuadrantReconciler) reconcileLimitador(ctx context.Context, kObj *kuadr
 }
 
 func (r *KuadrantReconciler) reconcileAuthorino(ctx context.Context, kObj *kuadrantv1beta1.Kuadrant) error {
-	tmpFalse := false
-	authorino := &authorinov1beta1.Authorino{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "Authorino",
-			APIVersion: "operator.authorino.kuadrant.io/v1beta1",
-		},
-		ObjectMeta: metav1.ObjectMeta{
-			Name:      "authorino",
-			Namespace: kObj.Namespace,
-		},
-		Spec: authorinov1beta1.AuthorinoSpec{
-			ClusterWide:            true,
-			SupersedingHostSubsets: true,
-			Listener: authorinov1beta1.Listener{
-				Tls: authorinov1beta1.Tls{
-					Enabled: &tmpFalse,
+	authorinoKey := client.ObjectKey{Name: common.AuthorinoName, Namespace: kObj.Namespace}
+	authorino := &authorinov1beta1.Authorino{}
+	err := r.Client().Get(ctx, authorinoKey, authorino)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			tmpFalse := false
+			authorino = &authorinov1beta1.Authorino{
+				TypeMeta: metav1.TypeMeta{
+					Kind:       "Authorino",
+					APIVersion: "operator.authorino.kuadrant.io/v1beta1",
 				},
-			},
-			OIDCServer: authorinov1beta1.OIDCServer{
-				Tls: authorinov1beta1.Tls{
-					Enabled: &tmpFalse,
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      common.AuthorinoName,
+					Namespace: kObj.Namespace,
 				},
-			},
-		},
+				Spec: authorinov1beta1.AuthorinoSpec{
+					ClusterWide:            true,
```

Comment:
There aren't many use cases for allowing multiple cluster-wide instances and yet do not to expose `authConfigLabelSelectors`. I can think of some, but in general it's not recommended.

Reply:
This was part of the existing configuration. I made the assumption that the existing configuration was correct.